docs/content/platform/new_blocks.md

```diff
@@ -121,16 +121,17 @@ from backend.data.model import (
 
 from backend.data.block import Block, BlockOutput, BlockSchema
 from backend.data.model import CredentialsField
+from backend.integrations.providers import ProviderName
 
 
 # API Key auth:
 class BlockWithAPIKeyAuth(Block):
     class Input(BlockSchema):
         # Note that the type hint below is require or you will get a type error.
         # The first argument is the provider name, the second is the credential type.
-        credentials: CredentialsMetaInput[Literal['github'], Literal['api_key']] = CredentialsField(
-            provider="github",
-            supported_credential_types={"api_key"},
+        credentials: CredentialsMetaInput[
+            Literal[ProviderName.GITHUB], Literal["api_key"]
+        ] = CredentialsField(
             description="The GitHub integration can be used with "
             "any API key with sufficient permissions for the blocks it is used on.",
         )
```

Comment:
nit: we should replace these with code snippets with admonishments similar to the tests.md files (lmk what you think about that btw)

Reply:
Could you clarify what you mean?

Reply:
https://dev-docs.agpt.co/platform/contributing/tests/#example-focused-test-explanation

See the little + icons on the example? Those are what I’m referring to

Reply:
Whaaaat, how does that work?

Reply:
I’d check the import file and the our test docs page because the actual docs suck at explaining it

https://squidfunk.github.io/mkdocs-material/reference/admonitions/

Reply:
https://squidfunk.github.io/mkdocs-material/reference/annotations/

Reply:
#8946